src/main/java/org/broadinstitute/hellbender/tools/exome/BayesianHetPulldownCalculator.java

```diff
@@ -0,0 +1,671 @@
+package org.broadinstitute.hellbender.tools.exome;
+
+import com.google.common.annotations.VisibleForTesting;
+import htsjdk.samtools.*;
+import htsjdk.samtools.filter.DuplicateReadFilter;
+import htsjdk.samtools.filter.NotPrimaryAlignmentFilter;
+import htsjdk.samtools.filter.SamRecordFilter;
+import htsjdk.samtools.reference.ReferenceSequenceFileWalker;
+import htsjdk.samtools.util.IntervalList;
+import htsjdk.samtools.util.SamLocusIterator;
+import org.apache.commons.math3.analysis.integration.gauss.GaussIntegrator;
+import org.apache.commons.math3.analysis.integration.gauss.GaussIntegratorFactory;
+import org.apache.commons.math3.util.FastMath;
+import org.apache.logging.log4j.LogManager;
+import org.apache.logging.log4j.Logger;
+import org.broadinstitute.hellbender.exceptions.GATKException;
+import org.broadinstitute.hellbender.exceptions.UserException;
+import org.broadinstitute.hellbender.utils.*;
+import org.broadinstitute.hellbender.utils.param.ParamUtils;
+
+import java.io.File;
+import java.io.IOException;
+import java.util.*;
+import java.util.stream.Collectors;
+import java.util.stream.IntStream;
+
+/**
+ * A Bayesian heterozygous SNP pulldown calculator. Base qualities are taken into account
+ * to increase precision (see CNV-methods.pdf for details).
+ *
+ * TODO
```

Comment:
I understand, but these are still up-in-the-air leads and not quite issue-worthy. I could always remove them from the code and take personal notes instead. What is preferred?